lib/hmmlearn/hmm.py

```diff
@@ -359,9 +359,8 @@ class MultinomialHMM(BaseHMM):
     MultinomialHMM(algorithm='viterbi',...
     """
 
-    # TODO: accept the prior on emissionprob_ for consistency.
-    def __init__(self, n_components=1,
-                 startprob_prior=1.0, transmat_prior=1.0,
+    def __init__(self, n_components=1, startprob_prior=1.0,
+                 transmat_prior=1.0, *, emissionprob_prior=1.0,
```

Comment:
This also needs a changelog entry ("For MultinomialHMM, parameters after transmat_prior are now keyword-only.")